src/main/java/org/apache/commons/crypto/utils/NativeCodeLoader.java

```diff
@@ -231,7 +231,7 @@ public static String getVersion() {
                 .getResource("/META-INF/maven/org.apache.commons.crypto/commons-crypto/pom.properties");
         if (versionFile == null) {
             versionFile = NativeCodeLoader.class
-                    .getResource("/org/apache/commons/crypto/VERSION");
```

Comment:
Thank @britter . It seems the `VERSION` file is still under `org/apache/commons/crypto/`(resources/org/apache/commons/crypto/VERSION), so I think we should use `/org/apache/commons/crypto/VERSION`, I also tested with disable ` .getResource("/META-INF/maven/org.apache.commons.crypto/commons-crypto/pom.properties");`, did I miss anything?